src/controllers/FoodController.js

```diff
@@ -2,7 +2,16 @@ import FoodModel from '../models//FoodModel';
 class FoodController {
     addFood(request, response) {
         FoodModel.addFood(request.body).then((result) => {
-            response.status(200).json({ status: 'success', data: result });
+            response.status(200).json({ status: 'success', data: result[0] });
+        });
+    }
+
+    getFoodMenu(request, response) {
+        FoodModel.getFoodMenu().then((result) => {
+            if (result.length > 0) 
+                response.status(200).json({ status: 'success', data: result });
+            else
+                response.status(200).json({ status: 'success', data: 'No food menu available' });
```

Comment:
If the result is an array, I think it might be more useful for the case of no menu yet to be an empty array and not a string. That way it might be easier to manage the data on the front-end. This is just a suggestion though.